src/caa-cases/store/actions/caa-cases.actions.ts

```diff
@@ -0,0 +1,75 @@
+import { Action } from '@ngrx/store';
+import { CaaCases } from '../../models/caa-cases.model';
+
+export const LOAD_ASSIGNED_CASES = '[CAA CASES] Load Assinged Cases';
```

Comment:
@johnbenjamin-hmcts - I think we should probably fix up any misspellings of "assigned" (i.e. "assinged")

Reply:
Thank you @Daniel-Lam I have now corrected the spellings.